src/wasm/wasm-stack.cpp

```diff
@@ -2709,14 +2713,15 @@ StackInst* StackIRGenerator::makeStackInst(StackInst::Op op,
   ret->origin = origin;
   auto stackType = origin->type;
   if (origin->is<Block>() || origin->is<Loop>() || origin->is<If>() ||
-      origin->is<Try>()) {
+      origin->is<Try>() || origin->is<TryTable>()) {
```

Comment:
I think we can replace this condition with `Properties::isControlFlowStructure` perhaps? It's equivalent and I think it has the right meaning here.

Reply:
Done